improver_tests/calibration/reliability_calibration/test_AggregateReliabilityCalibrationTables.py

```diff
@@ -33,225 +33,160 @@
 import unittest
 
 import numpy as np
+import pytest
 from numpy.testing import assert_array_equal
 
 from improver.calibration.reliability_calibration import (
     AggregateReliabilityCalibrationTables as Plugin,
 )
-from improver.calibration.reliability_calibration import (
-    ConstructReliabilityCalibrationTables as CalPlugin,
-)
-from improver_tests.calibration.reliability_calibration.test_ConstructReliabilityCalibrationTables import (  # noqa: E501
-    Test_Setup,
-)
-
-
-class Test_Aggregation(Test_Setup):
-
-    """Test class for the Test_AggregateReliabilityCalibrationTables tests,
-    setting up cubes to use as inputs."""
-
-    def setUp(self):
-        """Create reliability calibration tables for testing."""
-
-        super().setUp()
-        reliability_cube_format = CalPlugin()._create_reliability_table_cube(
-            self.forecasts_grid, self.expected_threshold_coord
-        )
-        self.reliability_cube = reliability_cube_format.copy(data=self.expected_table)
-        self.different_frt = self.reliability_cube.copy()
-        new_frt = self.different_frt.coord("forecast_reference_time")
-        new_frt.points = new_frt.points + 48 * 3600
-        new_frt.bounds = new_frt.bounds + 48 * 3600
-
-        self.masked_reliability_cube = self.reliability_cube.copy()
-        masked_array = np.zeros(self.reliability_cube.shape, dtype=bool)
-        masked_array[:, :, 0, :2] = True
-        self.masked_reliability_cube.data = np.ma.array(
-            self.reliability_cube.data, mask=masked_array
-        )
-
-        self.masked_different_frt = self.different_frt.copy()
-        masked_array = np.zeros(self.different_frt.shape, dtype=bool)
-        masked_array[:, :, :2, 0] = True
-        self.masked_different_frt.data = np.ma.array(
-            self.different_frt.data, mask=masked_array
-        )
-
-        self.overlapping_frt = self.reliability_cube.copy()
-        new_frt = self.overlapping_frt.coord("forecast_reference_time")
-        new_frt.points = new_frt.points + 6 * 3600
-        new_frt.bounds = new_frt.bounds + 6 * 3600
-
-        self.lat_lon_collapse = np.array(
-            [
-                [0.0, 0.0, 1.0, 2.0, 1.0],
-                [0.0, 0.375, 1.5, 1.625, 1.0],
-                [1.0, 2.0, 3.0, 2.0, 1.0],
-            ]
-        )
 
 
-class Test__repr__(unittest.TestCase):
```

Comment:
To aid other reviewers @tjtg, are the only tests that have been removed are the repr tests?

Reply:
Yes - only the repr tests have been removed. All other tests should be one-for-one equivalent.